example-get-started-experiments/code/.github/workflows/dvc-studio.yml

```diff
@@ -90,7 +90,7 @@ jobs:
       - uses: aws-actions/configure-aws-credentials@v1
         with:
           aws-region: us-east-2
-          role-to-assume: arn:aws:iam::342840881361:role/SandboxUser
+          role-to-assume: ${{ secrets.AWS_ROLE_TO_ASSUME }}
```

Comment:
do we have to generalize it in this way? (an extra step for us to take care of)

Reply:
It was suggested by @jesper7 and @tapadipti to use the role as a secret. I assume it has some security implications although I am not sure what is the danger of leaking a role name

Reply:
@daavoo How did you set it up for https://github.com/iterative/example-get-started-experiments? Should we add it to the readme here?

Reply:
Note that roles can be public.